Makefile

```diff
@@ -32,7 +32,7 @@ build: module-check ## Build site with non-production settings and put deliverab
 	hugo --minify --environment development
 
 build-preview: module-check ## Build site with drafts and future posts enabled
-	hugo --buildDrafts --buildFuture --environment preview
+	hugo --buildDrafts --buildFuture --environment preview --panicOnWarning
```

Comment:
no. I don't think we should go this far. It is too strict. Not all warnings are critical.

Reply:
@tengqm this PR is for a not-yet-accepted issue. Let's move the discussion [there](https://github.com/kubernetes/website/issues/34357).